### DISCUSSION TITLE
Is there any issue with table tags and virtualization?

### DISCUSSION THREAD

**Discussion by noghartt** [TanStack/virtual#365] [Q&A]
I follow some examples to put my `<tr>`s as virtualized items but trying it, my cells loses the reference from each column and creates weird styles with custom widths and other things.

**Comment by wuarmin** [comment]
Check this out:
https://tanstack.com/table/v8/docs/examples/react/virtualized-rows

I use it with react-table v7 and it works like a charm.

**Reply by noghartt** [reply]
I'm on react-table v8 and react-virtual v3, but it's breaking these styles and losing the reference from each column.

**Reply by wuarmin** [reply]
Do you have a sandbox showing your issue? 
Can you replicate it based on this sandbox: https://codesandbox.io/s/react-virtual-only-one-item-is-rendered-at-first-bfoj5r?file=/src/main.tsx
